packages/k-ramel/src/createStore/createStore.js

```diff
@@ -28,9 +28,32 @@ export default (definition, options = defaultOptions) => {
     },
   }
   const { init, hideRedux, drivers } = innerOptions
+  const definitionWithDrivers = { ...definition }
+
+  // use drivers
+  const driversEnhancers = []
+  const driversInits = []
+  Object.values(drivers)
+    .forEach((driver) => {
+      // bind reducer to store definition
+      if (driver.getReducer) {
+        const { reducer, path } = driver.getReducer() // eslint-disable-line no-unused-vars
```

Comment:
If someone find a way to do this a better way (and without using lodash)

Reply:
I don't know if it's a better way, but you don't have to use `eval` or  `lodash` :
```js
const reducerPath = path.split('.')
const reducerName = reducerPath.pop()
const reducerLocation = reducerPath.reduce((acc, next) => acc[next], definitionWithDrivers)
reducerLocation[reducerName] = reducer
```

Reply:
I have to benchmark that

Reply:
https://github.com/alakarteio/k-ramel/issues/132